.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ on:
     - 'release/*'
   pull_request:
 env:
-  CORE_REPO_SHA: 042d7a7921e25936decd95addf4fb1d30afb88e2
+  CORE_REPO_SHA: f5fb6b1353929cf8039b1d38f97450866357d901
```

Comment:
Reference to this change:
https://github.com/open-telemetry/opentelemetry-python/pull/3115